Fix CPU (short) quota description

rick-man opened the pull request:
Monitoring guidelines currently describe the CPU Time (Short) quota as the CPU quota per three-minute window. According to [App Service Limits](https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#app-service-limits), however, it is the CPU quota per five-minute window. One of these two docs is incorrect.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Blackmist commented:
Changes look good. Will work with the code author on suggested changes that might impact code (cotage vs. cottage, data dataframe, etc.)

Thanks!

rick-man committed:
Fix CPU (short) quota description

Monitoring guidelines currently describe the CPU Time (Short) quota as the CPU quota per three-minute window. According to [App Service Limits](https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#app-service-limits), however, it is the CPU quota per five-minute window. One of these two docs is incorrect.
